modules/op-results-db.adoc

```diff
@@ -0,0 +1,34 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-tekton-results-for-openshift-pipelines-observability.adoc
+
+:_content-type: PROCEDURE
+[id="results-db_{context}"]
+= Preparing a secret with the database credentials
+
+{tekton-results} uses a PostgreSQL database to store data. You can configure the installation to use either a PostgreSQL server that is automatically installed with {tekton-results} or an external PostgreSQL server that already exists in your deployment. In both cases, provide a secret with the database credentials.
+
+.Procedure
+
+Complete one of the following steps:
```

Comment:
This would look like a single bullet with two double-bullets under it and then no second single-bullet. I suspect that it might confuse the reader, who might think there should have been a second single-bullet?

Reply:
That's fair. I've wrestled with similar in the past, and haven't found a for-sure mod-docs call on it yet. 😅